data/data-pipeline/data_pipeline/tests/score/test_output.py

```diff
@@ -1,12 +1,39 @@
-# flake8: noqa: W0613,W0611,F811
+# flake8: noqa: W0613,W0611,F811,
+# pylint: disable=unused-import,R0913
+import inspect
 from dataclasses import dataclass
+from re import I
 from typing import List
 import pytest
 import pandas as pd
+import numpy as np
 from data_pipeline.score import field_names
-from .fixtures import final_score_df  # pylint: disable=unused-import
+from .fixtures import (
+    final_score_df,
+    ejscreen_df,
+    hud_housing_df,
+    census_df,
+    cdc_places_df,
+    census_acs_median_incomes_df,
+    cdc_life_expectancy_df,
+    doe_energy_burden_df,
+    national_risk_index_df,
+    dot_travel_disadvantage_df,
+    fsf_fire_df,
+    nature_deprived_df,
+    eamlis_df,
+    fuds_df,
+    geocorr_urban_rural_df,
+    census_decennial_df,
+    census_2010_df,
+    hrs_df,
+    national_tract_df,
+)
+
 
 pytestmark = pytest.mark.smoketest
+GEOID_TRACT_FIELD_NAME = field_names.GEOID_TRACT_FIELD
+UNMATCHED_TRACK_THRESHOLD = 1000
```

Comment:
I don't think I understand the question --- could you explain more of what you're thinking?

Reply:
yeah! why not just access GEOID fieldname as `fixtures.GEOID...`

Reply:
OHHH! The reason I defined `GEOID_TRACT_FIELD_NAME = field_names.GEOID_TRACT_FIELD` at the top of the file was because I knew I'd be referencing it a TON of times and honestly just wanted to save the namespace. BUT, now that I look at it again, there's a more obvious way to do that in python, so I did that in da3554de50cf56d627ac896d9ae1e92a3816f727

Reply:
OHHH! The reason I defined GEOID_TRACT_FIELD_NAME = field_names.GEOID_TRACT_FIELD at the top of the file was because I knew I'd be referencing it a TON of times and honestly just wanted to save the namespace. BUT, now that I look at it again, there's a more obvious way to do that in python, so I did that in 2fac8a1ebb02a26fa9f189deccad0a08fabecf15

Reply:
do we actually mean `TRACT` not `TRACK`?

Reply:
Yes, I did :D